WordPress/Classes/Networking/ReaderTopicServiceRemote.m

```diff
@@ -2,7 +2,9 @@
 #import "WordPressComApi.h"
 #import "RemoteReaderTopic.h"
 #import "ReaderTopic.h"
+#import "RemoteReaderSiteInfo.h"
 
+NSString * const WordPressComReaderEndpointURL = @"https://public-api.wordpress.com/rest/v1.2/";
```

Comment:
Maybe we should move this URL to `Constants.h / m` maybe? (i'd assume the whole app should end up hitting the REST 1.2 endpoint right?